ntbs-service/Pages/_HomepageKpiDetails.cshtml

```diff
@@ -0,0 +1,34 @@
+﻿@model ntbs_service.Pages.IndexModel
+
+<div>
+    @if (@Model.HomepageKpiDetails.Any())
+    {
+        <h1 hidden> Homepage KPI Details </h1>

```

Comment:
this wants to be `class="govuk-visually-hidden"` which will allow it to be read by SRs